tests/lib/rules/strict-component-boundaries.js

```diff
@@ -25,6 +25,10 @@ ruleTester.run('strict-component-boundaries', rule, {
       code: `import {someThing} from '../OtherComponent';`,
       parserOptions,
     },
+    {
+      code: `import someThing from '../OtherComponent/fixtures/mock-query.json';`,
```

Comment:
I still think this should be caught, but `../fixtures/OtherComponent/foo.json` should not (this means reaching up into another component, and getting its fixture, which doesn't seem right).